ApplyToTheIndustry/Assets/Scripts/Skills/SkillGroup.cs

```diff
@@ -1,40 +1,57 @@
 using System.Collections.Generic;
+using UnityEngine;
 
-public class SkillGroup
+public class SkillGroup: MonoBehaviour
```

Comment:
Not sure I agree with this being a Monobehavior - I would likely make it a ScriptableObject if we need to configure it inspector.